templates/home/api.html

```diff
@@ -21,121 +21,147 @@
 </dl>
 
 <h2>Technical specification</h2>
 
-<h3>Home page banner</h3>
+<h3>Your government</h3>
 
 <dl>
 	<dt>Version</dt>
-	<dd>2</dd>
+	<dd>3.0</dd>
 	<dt>Updated</dt>
-	<dd>2019-05-09</dd>
+	<dd>2022-04-27</dd>
 </dl>
 
-<p>The background image is loaded with the provisional plugin bgimg which set the background image. The background image is hosted in the WMS at this location. <code>https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg</code>.</p>
-
-<p>The intend is to publish background image that represent natural Canadian landscape and may get updated montly or seasonal.</p>
-
-<h4>Previous version</h4>
+<p>There are 8 links in this section presented in 2 columns.</p>
+<p>In the previous version, there were 4 link only.</p>
+
+<div class="row">
+	<div class="col-md-6">
+		<h3>Version 3.0</h3>
+		<pre>
+			<code>
+				&lt;section class="home-your-gov gc-srvinfo well well-sm brdr-0"&gt;
+					&lt;div class="container"&gt;
+						&lt;h2&gt;Your government&lt;/h2&gt;
+						&lt;div class="row mrgn-bttm-lg"&gt;
+							&lt;div class="col-lg-7 col-xs-12 col-md-6"&gt;
+								<ins>
+									&lt;ul class="row wb-eqht-grd list-unstyled small"&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://www.canada.ca/en/contact.html"&gt;Contact us&lt;/a&gt;
+										&lt;/li&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://www.canada.ca/en/news.html"&gt;News&lt;/a&gt;
+										&lt;/li&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://pm.gc.ca/en"&gt;Prime Minister&lt;/a&gt;
+										&lt;/li&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://www.canada.ca/en/government/dept.html"&gt;Departments and agencies&lt;/a&gt;
+										&lt;/li&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://www.canada.ca/en/government/system.html"&gt;About government&lt;/a&gt;
+										&lt;/li&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://open.canada.ca/en"&gt;Open government and data&lt;/a&gt;
+										&lt;/li&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://www.canada.ca/en/government/publicservice.html"&gt;Working for the government&lt;/a&gt;
+										&lt;/li&gt;
+										&lt;li class="col-xs-12 col-sm-6 mrgn-tp-sm mrgn-bttm-sm"&gt;
+											&lt;a href="https://www.canada.ca/en/government/system/laws.html"&gt;Treaties, laws and regulations&lt;/a&gt;
+										&lt;/li&gt;
+									&lt;/ul&gt;
+								</ins>
+							&lt;/div&gt;
+						&lt;/div&gt;
+					&lt;/div&gt;
+				&lt;/section&gt;
+			</code>
+		</pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Before version 3.0</h3>
```

Comment:
```suggestion
		<h3>Deprecated - Version 2.0</h3>
```

Reply:
Suggested change applied but kept version 2 as suggested in your comments below. Thanks